.github/workflows/node-when-unrelated.yml

```diff
@@ -0,0 +1,43 @@
+# This workflow is provided via the organization template repository
+#
+# https://github.com/nextcloud/.github
+# https://docs.github.com/en/actions/learn-github-actions/sharing-workflows-with-your-organization
+#
+# Use node together with node-when-unrelated to make eslint a required check for GitHub actions
+# https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/troubleshooting-required-status-checks#handling-skipped-but-required-checks
+
+name: Node
+
+on:
+  pull_request:
+    paths-ignore:
+      - '.github/workflows/**'
+      - 'src/**'
```

Comment:
Should it not be `lib/**`?

Reply:
It's from the template :shrug:

Reply:
I guess it doesn't fully apply to some npm packages :shrug:

Reply:
So written for apps not libs, but I think it should work by the last few rules so